piplicenses.py

```diff
@@ -247,6 +254,20 @@ def create_summary_table(args):
     return table
 
 
+def flatten_single_entry_lists(rows):
```

Comment:
This function can be defined as an instance method of the CSVPrettyTable class.

e.g. )
```python
class CSVPrettyTable(PrettyTable):
    def _flatten_single_entry_lists(self, rows):
```

Or you can define it as a function in the scope in `def get_string: `

Defining it in global scope is excessive.

Reply:
Sure, can do. I wasn't sure if we were planning to try to flatten the output for any other formats or just CSV.